crates/primitives/Cargo.toml

```diff
@@ -92,11 +97,20 @@ toml = "0.7.4"
 # https://github.com/paradigmxyz/reth/pull/177#discussion_r1021172198 
 secp256k1.workspace = true
 criterion = "0.5"
-pprof = { version = "0.12", features = ["flamegraph", "frame-pointer", "criterion"] }
+pprof = { version = "0.12", features = [
+    "flamegraph",
+    "frame-pointer",
+    "criterion",
+] }
 
 [features]
 default = []
-arbitrary = ["revm-primitives/arbitrary", "dep:arbitrary", "dep:proptest", "dep:proptest-derive"]
+arbitrary = [
+    "revm-primitives/arbitrary",
+    "dep:arbitrary",
+    "dep:proptest",
+    "dep:proptest-derive",
+]
```

Comment:
Can you keep formatting the same as before?

Reply:
yes, sorry I have a fixed config on rust-analyzer (the one suggested in the README file) and it automatically adjusted it. I will revert it as it was before